### ISSUE TITLE
Delegating loading variables

### ISSUE THREAD

**Issue by flaviocordova** [samtgarson/nuxt-env#6]
The current configuration is based on reading values in process.env based on their keys.
However, I need to add some extra logic (like assuming defaults, for exemple) and there's no way to do that...

what do you guys think adding an extra way to configure the env that would delegate the loading, something like this:

```
// nuxt.config.js

// Tell nuxt-env which env vars you want to inject
modules: ['nuxt-env', {
  keys: ['TEST_VALUE'],
  properties: require('my-env-logic')
}]
```

**Comment by samtgarson** [comment]
Hmm I think that might add complexity here and potentially inefficiency? Defaults seems like a legit need though, how about:

```js
module: ['nuxt-env', {
  keys: [
    'TEST_VALUE', 
    { key: 'ANOTHER_VALUE', default: 'DEFAULT_VALUE'} 
  ]
}]
```

If you need to delegate the entire module, you could just do `keys: require('./env-keys')`?

**Comment by ruudboon** [comment]
Looking for this also. Setting a default like this would be perfect.

**Comment by robsontenorio** [comment]
Is there any solution to set default values?

**Comment by samtgarson** [comment]
Defaults released in `0.1.0` 🚀
